### PR TITLE
[CWS] add chdir event

### PR DESCRIPTION
**Pull request by safchain**
### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided. Except if the `qa/skip-qa` label, with required either `qa/done` or `qa/no-code-change` labels, are applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by spikat — APPROVED** [review]

File: docs/cloud-workload-security/agent_expressions.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ Triggers are events that correspond to types of activity seen by the system. The
 | `bind` | Network | A bind was executed | 7.37 |
 | `bpf` | Kernel | A BPF command was executed | 7.33 |
 | `capset` | Process | A process changed its capacity set | 7.27 |
+| `chdir` | File | [Experimental] A process change the current directory | 7.52 |
```
Comment (hestonhoffman) [inline docs/cloud-workload-security/agent_expressions.md:38]:
```suggestion
| `chdir` | File | [Experimental] A process changed the current directory | 7.52 |
```

File: docs/cloud-workload-security/agent_expressions.md [hunk 2]
```diff
@@ -435,6 +436,36 @@ A process changed its capacity set
 | [`capset.cap_effective`](#capset-cap_effective-doc) | Effective capability set of the process |
 | [`capset.cap_permitted`](#capset-cap_permitted-doc) | Permitted capability set of the process |
 
+### Event `chdir`
+
+_This event type is experimental and may change in the future._
+
+A process change the current directory
```
Comment (hestonhoffman) [inline docs/cloud-workload-security/agent_expressions.md:443]:
```suggestion
A process changed the current directory
```

**Review by hestonhoffman — COMMENTED** [review]
Small edit

**Review by YoannGh — APPROVED** [review]

**Pull request merged**